codegen/smithy-aws-swift-codegen/src/test/kotlin/software/amazon/smithy/aws/swift/codegen/TestContextGenerator.kt

```diff
@@ -36,7 +36,6 @@ class TestContextGenerator {
             val service = model.getShape(ShapeId.from(serviceShapeIdWithNamespace)).get().asServiceShape().get()
             val settings = buildDefaultSwiftSettingsObjectNode(serviceShapeIdWithNamespace)
             val swiftSettings = SwiftSettings.from(model, settings)
-            model = AddOperationShapes.execute(model, swiftSettings.getService(model), swiftSettings.moduleName)
```

Comment:
This is not necessary and is inconsistent with how this is performed in release.
By calling this here, we end up adding operation shapes before any of the integrations are executed. In release, we execute the integrations first, then add our operation shapes. By removing this line, we make our testing environment more consistent with our releases.

This logic is perform within `SwiftCodegenPlugin().execute(pluginContext)` which occurs a few lines below this.